### PR TITLE
Update to use GitHub Actions for CI

### PR DESCRIPTION
**Pull request by owenvoke**
This updates Collision to use GitHub Actions for CI instead of Travis CI.

### CODE REVIEW

File: .github/workflows/formats.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: [7.4]
```
Comment (nunomaduro) [inline .github/workflows/formats.yml:13]: 7.3 missing here.

**Comment by nunomaduro** [comment]
@owenvoke Once tests are passing it can be merged. Going to add you as maintainer.

**Comment by owenvoke** [comment]
@nunomaduro, just wondering about the failing formatter tests.

It's basically failing due to the "risky" `declare_strict_types` PHPCS rule. Should we add `->setRiskyAllowed(true)` to the `.php_cs`, or just add `--allow-risky` in CI? 🙂

**Comment by nunomaduro** [comment]
--allow-risky=yes on the CI

**Comment by owenvoke** [comment]
Right, formatting is passing. 👍 Will investigate the tests and try to get this sorted asap.

Looks like they're failing due to CRLF line endings, but I have no idea why. 🤔

**Comment by nunomaduro** [comment]
Thank you!

**Pull request merged**